docs/roles/system_manager/manage_deployments.md

```diff
@@ -6,6 +6,32 @@
 This document assumes that you already have access to a {ref}`Safe Haven Management (SHM) environment <deploy_shm>` and one or more {ref}`Secure Research Environments (SREs) <deploy_sre>` that are linked to it.
 ```
 
+(add_resize_vm)=
+
+## {{heavy_plus_sign}} Add a new Secure Research Desktop (SRD) or resize an existing SRD’s VM
```

Comment:
Any reason for these two things to be one section?

I think the two operations are conceptually quite different (even if they might use the similar commands).

Reply:
I agree, I'll separate these out